packages/core/src/services/documents/destroyDocument.ts

```diff
@@ -0,0 +1,39 @@
+import { database, Database } from '$core/client'
+import { NotFoundError, Result, Transaction } from '$core/lib'
+import { DocumentVersionsRepository } from '$core/repositories'
+import { Commit, DocumentVersion } from '$core/schema'
+import { destroyOrSoftDeleteDocuments } from '$core/services/documents/destroyOrSoftDeleteDocuments'
+import { assertCommitIsDraft } from '$core/services/documents/utils'
+
+export async function destroyDocument({
+  document,
+  commit,
+  workspaceId,
+  db = database,
+}: {
+  document: DocumentVersion
+  commit: Commit
+  workspaceId: number
```

Comment:
This is not ideal as I [commented here](https://github.com/latitude-dev/latitude-llm/pull/60/files#r1693971751) a commit passed here was found with a repository so the workspace was already there. 

There has to be a nice way of having `commit.workspace.id` so object includes relations already resolved

Reply:
just pass a workspace to this method instead of workspaceId